### PR TITLE
Add Comms Shadows contact info for v. 1.20

### PR DESCRIPTION
**Pull request by jrsapi**
**What type of PR is this:**

/kind documentation

**What this PR does / why we need it:**

Add 1.20 Comms shadows

**Which issue(s) this PR fixes:**

None
Special notes for your reviewer:

/assign @jeremyrickard @savitharaghunathan @hasheddan @palnabarun @lachie83

### CODE REVIEW

File: releases/release-1.20/release_team.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
...
 | Docs | Anna Jung ([@annajung](https://github.com/annajung) / Slack: `@annajung`) | Leslie-Alexandre Denis ([@eagleusb](https://github.com/eagleusb) / Slack: `@grumpycat`), Rey Lejano ([@reylejano-rxm](https://github.com/reylejano-rxm) / Slack: `@rey.lejano`), Kristin Martin ([@kcmartin](https://github.com/kcmartin) / Slack: `@kcmartin`), Somtochi Onyekwere ([@SomtochiAma](https://github.com/SomtochiAma) / Slack: `@SomtochiAma`)  |
 | Release Notes | James Laverack ([@JamesLaverack](https://github.com/JamesLaverack) / Slack: `@james.laverack`) | |
-| Communications | Joseph Sandoval ([@jrsapi](https://github.com/jrsapi) / Slack: `@j-dawg`) | |
+| Communications | Joseph Sandoval ([@jrsapi](https://github.com/jrsapi) / Slack: `@j-dawg`) | Divya Mohan ([@divya-mohan0209](https://github.com/divya-mohan0209) /  Slack: @divya), Shubham Tatvamas ([@shubhamtatvamasi](https://github.com/ShubhamTatvamasi) / Slack: @Shubham Tatvamasi ), Tony Gosselin ([@sfotony](https://github.com/sfotony) / Slack: @sfotony ), Chris Short [(@chris-short)](https://github.com/chris-short) / @chrisshort ) | |
```
Comment (hasheddan) [inline releases/release-1.20/release_team.md:11]: nitpick: I think we usually list alphabetically by last name :)

Comment (jrsapi) [reply releases/release-1.20/release_team.md:11]: i'll reorder by last name.

**Review by savitharaghunathan — COMMENTED** [review]
/lgtm

**Comment by savitharaghunathan** [comment]
/hold

@jrsapi Can you squash the commits into one, please?

**Review by savitharaghunathan — COMMENTED** [review]
/lgtm

**Comment by jrsapi** [comment]
Commits squashed. Thanks both of you for prompt feedback.

**Comment by savitharaghunathan** [comment]
/hold cancel
Thanks for squashing the commits, @jrsapi :)

**Pull request merged**